i18n/en.toml

```diff
@@ -610,20 +610,37 @@ started = "Started"
 title = "Charging Session"
 
 [sessions]
+avgCo2TitleByGroup = "⌀ CO₂ {byGroup}"
 avgPower = "⌀ Power"
 avgPrice = "⌀ Price"
+avgPriceTitleByGroup = "⌀ Price {byGroup}"
+byGroupLoadpoint = "by Charging Point"
+byGroupVehicle = "by Vehicle"
 chargeDuration = "Duration"
 co2 = "⌀ CO₂"
 csvMonth = "Download {month} CSV"
 csvTotal = "Download total CSV"
 date = "Start"
 downloadCsv = "Download as CSV"
 energy = "Charged"
+energyGroupedTitle = "Charged Energy {energy}"
+energyGroupedTitleByGroup = "Energy {byGroup}"
+energySubTitle = "{energy} total"
+energyTitle = "{percent} Solar Energy"
+groupedCo2TitleByGroup = "CO₂-Amount {byGroup}"
+groupedPriceTitleByGroup = "Total Cost {byGroup}"
```

Comment:
Cost im Key

Reply:
Nope. Technisch ist bei uns "Cost" der Oberbegriff für "Co2" und "Preis" (siehe auch Smart Cost Limit). Wie wir das dann konkrete in der UI betiteln ist ein anderes Thema.